### PR TITLE
Added Toky

### PR DESCRIPTION
**Pull request by tokyapp**
Pull request for Toky API as Data Studio source

### CODE REVIEW

**Comment by tokyapp** [comment]
The CLA for this request was already signed :-)

File: sources/toky_api.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": 		"TOKY_API",
+  "name": 		"Toky Call Data API",
```
Comment (pfrisella) [inline sources/toky_api.json]: This should describe the data source, not the method in which it is obtained (i.e. not API). So I suggest just using Toky Call Data.

Also update the Id to be either TOKY or if you expect to have other data sources in the future then be more specific TOKY_CALL_DATA. You should also update the filename once you update the Id.

File: sources/toky_api.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "categories": 	["ANALYTICS", "TOOLS"],
```
Comment (pfrisella) [inline sources/toky_api.json]: Does it also make sense to use CALL_TRACKING instead of TOOLS?

File: sources/toky_api.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "organization": 	"TOKY",
+  "iconUrl": 		"https://s3.amazonaws.com/tokystorage/images/toky-48x48.png",
+  "sourceUrl": 		"http://developer.toky.co/",
```
Comment (pfrisella) [inline sources/toky_api.json]: This link is to help users learn more about the source and service, not the method/API from which it was fetched. Is there a more appropriate URL so users can learn more bout Toky Call Data?

**Comment by tokyapp** [comment]
Changes made. Thank you, @pfrisella 👍.

**Pull request merged**